pkg/core/dao_test.go

```diff
@@ -5,9 +5,9 @@ import (
 
 	"github.com/CityOfZion/neo-go/pkg/core/state"
 	"github.com/CityOfZion/neo-go/pkg/core/storage"
-	"github.com/CityOfZion/neo-go/pkg/core/testutil"
 	"github.com/CityOfZion/neo-go/pkg/core/transaction"
 	"github.com/CityOfZion/neo-go/pkg/crypto/keys"
+	"github.com/CityOfZion/neo-go/pkg/internal"
```

Comment:
Are you sure it's worth using `internal` and not `internal/testrandom` or something?

Reply:
`internal/random`

usage:
`random.String()`